iceoryx_posh/cmake/iceoryx_posh_testingConfig.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Did you write this file yourself or was it copy and pasted?

Reply:
no, it was copied but it seems git did not recognize it properly. It's the same file like iceoryx_posh/cmake/iceoryx_poshConfig.cmake